package.json

```diff
@@ -109,6 +110,7 @@
     "tstyche": "1.0.0-beta.9",
     "tsx": "4.6.2",
     "typescript": "5.3.3",
+    "vitest": "1.2.1",
```

Comment:
Adding `vitest` here so that we can use it to run `server.test.mts` in tasks/server-tests